task-runner/task_runner/task_request_handler.py

```diff
@@ -348,6 +359,18 @@ def __call__(self, request: Dict[str, str]) -> None:
         # Catch all exceptions to ensure that we log the error message
         except Exception as e:  # noqa: BLE001
             message = utils.get_exception_root_cause_message(e)
+
+            output_size = self._pack_output()
+
+            self._publish_event(
+                events.TaskOutputUploaded(
+                    id=self.task_id,
+                    machine_id=self.executer_uuid,
+                    new_status=task_status.TaskStatusCode.FAILED.value,
+                    output_size=output_size,
+                    terminal=False,
+                ))
```

Comment:
The method `save_output` should be called instead.

Reply:
Thanks, updated.